webextensions/api/browsingData.json

```diff
@@ -560,7 +571,7 @@
                 "version_added": "53"
               },
               "firefox_android": {
-                "notes": "See <a href='https://bugzil.la/1363010'>bug 1363010</a>. <a href='https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/remove'><code>browser.history.remove(options, {history:true})</code></a> can be used instead.",
+                "notes": "See <a href='https://bugzil.la/1363010'>bug 1363010</a>. <a href='https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/remove'><code>browser.history.remove(options, {history:true})</code></a> can be used up to Firefox for Android 79 instead.",
                 "version_added": false
               },
```

Comment:
Can't comment directly, but removeLocalStorage a few line below this has a note that says that the API is defined by returns a rejected promise which is not accurate anymore.

Reply:
Also the note for removePasswords is outdated too, as that bug has been closed.